dockerfiles/Dockerfile.openshift

```diff
@@ -1,43 +1,42 @@
-# FROM directive instructing base image to build upon
-# This could be used as a base instead: 
-# https://hub.docker.com/r/nikolaik/python-nodejs
-FROM python:3.8-slim
+# node-build stage
+
+FROM docker-registry.default.svc:5000/openshift/node:16-slim AS node-build
+WORKDIR /build/
+
+COPY frontend .
+RUN npm install
+
+RUN npm run build:frontend
+
+# main stage
+
+FROM docker-registry.default.svc:5000/openshift/python:3.8-slim AS main
```

Comment:
I didn't test pulling from the OpenShift registry, I just removed the namespace to pull from Docker Hub. Seems to work though, should be fine with deployments.

Reply:
When I started this last year I wanted to see if there was a better way to avoid multi-stage builds, but as the year went on and we looked to finally be moving off Openshift 3, as well as Github Container Registry, it having mutlstage just was less of a problem. I think we can actually consolidate these files once we get to ROSA.